Dockerfile

```diff
@@ -6,27 +6,20 @@ RUN cd /tmp && \
     wget https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz && \
     tar -xf go1.12.7.linux-amd64.tar.gz && \
     mv go /usr/local
-WORKDIR /go/src/github.com/prebid/prebid-server/
+RUN mkdir -p /app/prebid-server/
+WORKDIR /app/prebid-server/
 ENV GOROOT=/usr/local/go
-ENV GOPATH=/go
-ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH
-RUN apt-get update && \
-    apt-get install -y git go-dep && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ENV PATH=$GOROOT/bin:$PATH
 ENV CGO_ENABLED 0
 COPY ./ ./
-RUN dep ensure && \
-    go build .
+RUN go build -mod=vendor .
 
 FROM ubuntu:18.04 AS release
 LABEL maintainer="hans.hjort@xandr.com" 
 WORKDIR /usr/local/bin/
-COPY --from=build /go/src/github.com/prebid/prebid-server/prebid-server .
+COPY --from=build /app/prebid-server/ .
 COPY static static/
 COPY stored_requests/data stored_requests/data
-RUN apt-get update && \
-    apt-get install -y ca-certificates mtr && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Sorry, I didn't catch this earlier but any reason to remove the `ca-certificates` and `mtr` package installation? I am not a 100% sure if we even need it in the first place. Likely not and therefore this should be fine but I just wanna confirm. Also, we should still at least keep the  `apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*` clean up command.

Reply:
LGTM otherwise. Thanks @benjaminch :)

Reply:
No worries, let's keep it all then, no real reasons here :)